packages/flutter/lib/src/material/time.dart

```diff
@@ -112,6 +112,30 @@ class TimeOfDay {
     );
   }
 
+  /// Whether this [TimeOfDay] occurs earlier than [other].
+  bool isBefore(TimeOfDay other) => compareTo(other) < 0;
+
+  /// Whether this [TimeOfDay] occurs later than [other].
+  bool isAfter(TimeOfDay other) => compareTo(other) > 0;
+
+  /// Whether this [TimeOfDay] occurs at the same time as [other].
```

Comment:
nit: add an explanation that this will be imprecise with everything besides hour and minute. So different days and seconds will be ignored. It might be overkill to call it out with how `TimeOfDay` works but I'm nervous about overpromising with anything to do with DateTime.